test/orderController.test.js

```diff
@@ -0,0 +1,137 @@
+const { app, connectDB } = require('../src/expressServer'); // Import your Express app here
+const { StatusCodes } = require('http-status-codes');
+const request = require('supertest');
+const { MongoMemoryReplSet } = require('mongodb-memory-server');
+const User = require('../src/models/User');
+const Album = require('../src/models/Album');
+const { loginAndReturnCookie } = require('./test_helper');
+
+let server;
+let mongooseConnection;
+let mongodb;
+
+jest.mock('stripe', () => {
+  // When `require('stripe')` is called, the below jest mock function will be returned instead
+  return jest.fn(() => ({
+    // When the `require('stripe')(API_KEY)` is called, the below object will be returned instead
+    paymentIntents: {
+      // The `stripe.paymentIntents.create` will be replaced with the below jest mock function that returns our mock data
+      create: jest.fn(() => ({
+        id: 'pi_mock123',
+        client_secret: 'mock_secret',
+      })),
+    },
+  }));
+});
+
+beforeAll(async () => {
+  mongodb = await MongoMemoryReplSet.create({
+    replSet: { storageEngine: 'wiredTiger' },
+  });
```

Comment:
Might be worth adding a comment here for future readers for why we had to use this different mongo server class and storage engine. Something about "A version of the mongo server that supports replica sets must be used for the order controller test cases because Mongo transactions are used"